tasks/main.yml

```diff
@@ -20,7 +20,9 @@
 - name: Set variable `timesync_services` with filtered uniq service names
   set_fact:
     timesync_services: "{{ ansible_facts.services.values() |
-          selectattr('name', 'defined') | map(attribute='name') |
+          selectattr('name', 'defined') |
+          rejectattr('status', 'match', '^not-found$') |
```

Comment:
Looking at the fix in the service_fact module upstream, https://github.com/ansible/ansible/blob/stable-2.12/lib/ansible/modules/service_facts.py#L212, I think the following must be added also:
*  `rejectattr('status', 'match', '^masked$')`
* `rejectattr('status', 'match', '^failed$')`
Same workaround is required here:
https://github.com/linux-system-roles/storage/blob/master/tasks/main-blivet.yml#L54
I've opened AAP-1128 on Red Hat's Jira to cherry-pick this to ealrier Ansibles.